robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -154,14 +154,27 @@ static void rq_state_start_stream()
  */
 static void rq_state_run()
 {
+	static unsigned char retries = 0;
+
 	rq_com_listen_stream();
 
 	if(rq_com_get_valid_stream() == false)
 	{
+		retries++;
+		printf("invalid stream : retry #%d\n", retries);
+
+		if (retries >= 100)
```

Comment:
to the print instruction: It should not be a print, but in this file there is no ROS. It's more a library which is then wrapped into a ros node afterwards. So in this file, roslogging is not available, but the output might be worth piping to ros anyway...